modules/nextflow/src/main/groovy/nextflow/k8s/model/PodOptions.groovy

```diff
@@ -66,6 +68,7 @@ class PodOptions {
     PodOptions( List<Map> options=null ) {
         int size = options ? options.size() : 0
         envVars = new HashSet<>(size)
+        mountCsiEphemerals = new HashSet<>(size)
         mountSecrets = new HashSet<>(size)
         mountConfigMaps = new HashSet<>(size)
         mountClaims = new HashSet<>(size)
```

Comment:
Setting the map size helps to allocate unneeded memory. Considering that this can be invoked tens of thousand of times is helps a bit to contain the garbage collection activity.